### ISSUE TITLE
Please continue with this project!

### ISSUE THREAD

**Issue by pmfederico** [mike168m/Cysboard#39]
I've seen the last modification was about 4 years ago, please go on! The community needs something like this instead of conky

**Comment by wakatara** [comment]
I have to agree. I was literally just looking for something like Ubersicht on OSX (and GeekTool) previously to ease the things I want to display on my i3 system and really need something that will display arbitrary, styled html on the root X Server or similar.

My C++ is pretty crap (and have never done any X programming) but would also be willing to try and help out on this now I'm back over on Linux.

**Comment by mik30s** [comment]
@pmfederico @wakatara 
I agree too and I'm so sorry I haven't kept up with my promises. Will definitely come back to this If I can.

**Comment by wakatara** [comment]
@mike168m 

Man, we're all busy and it's been a very, *very* strange 2020 for everyone. We all know how it is. I have not updated one of my own projects in 4 months despite wanting to add a couple of features into it. We'd all love to see this move forward though (lemme know if you feel like converting this to GoLang as then I could be of help here... or maybe Rust though still learning it so not too sure how "quality" my help would be there.).

If you're on OSX do take a look at Ubersicht. I feel it's got a similar model of what most people would want to do, though i think the first thing is just figure out how to display a css styled html page on X0 (and Weyland if we can get there though, I have no idea how the window system works there.). 

let us know how we can help. Sometimes work shared is work eased... 😃 

// @pmfederico
